src/snowflake/snowpark/_internal/ast.py

```diff
@@ -20,14 +25,27 @@ def expr_to_dataframe_expr(expr):
     return dfe
 
 
+def ast_expr_from_python_val(expr, val):
+    if isscalar(val):
+        expr.scalar = val
+    elif isinstance(val, DataFrame):
+        expr.dataframe_expr = val
+    elif isinstance(val, Series):
+        expr.series_expr = val
+    elif isinstance(val, slice):
+        expr.slice_expr = val
+    elif is_list_like(val):
+        expr.array_expr = val
+
+
 class AstBatch:
     def __init__(self, session):
         self._session = session
         self._id_gen = itertools.count(start=1)
         self._init_batch()
         # TODO: extended version from the branch snowpark-ir.
 
-    def assign(self, symbol = None):
+    def assign(self, symbol=None):
```

Comment:
Did the linter do this? I see spaces around this use case of `=` in most of our Python code.

Reply:
not sure, I'll fix this

Reply:
I think this is the linter